### PR TITLE
added the table to show the scores for students

### PR DESCRIPTION
**Pull request by naderakhgari**
Thank you for contributing! To help the review process, please provide the following:

### Proposal
added the table to show the scores for students

### Related
Provide links to any associated pull requests or issues. You can use `#123` to link to a PR or issue in this repository, or `user/repo#123` to link to other repositories.

### Checklist

- [ ] I have read the [code of conduct] and [contributing guide]
- [ ] I have made this pull request to the `master` branch
- [ ] I have run all of the automated validation using `npm run ship`
- [ ] I have added myself to the `"contributors"` list in the `package.json` (or do not want to)

[code of conduct]: https://github.com/textbook/starter-kit/blob/master/.github/CODE_OF_CONDUCT.md
[contributing guide]: https://github.com/textbook/starter-kit/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: client/src/Components/Results.js [hunk 1]
```diff
@@ -126,44 +130,179 @@ export default function Results(props) {
...
+		if (allAttempts.length <= 0) {
+			return 0;
+		}
+
+		allAttempts.length > attemptCounter
+			? setAttemptCounter(allAttempts.length)
+			: null;
+		let timestamp = getLastAttempt(student);
+		if (!timestamp) {
+			return null;
```
Comment (pendletons) [inline client/src/Components/Results.js:159]: Is it a problem to return `null` here instead of 0?

File: client/src/Components/Results.js [hunk 2]
```diff
@@ -126,44 +130,179 @@ export default function Results(props) {
...
-							return (
-								<option key={quiz._id} value={quiz._id}>
-									{quiz.name}
-								</option>
-							);
-						})}
-					</select>
+				<div className="col-12 quiz-handler">
+					<div className="col-6">
+						<h1>Welcome to QuizzTime</h1>
```
Comment (pendletons) [inline client/src/Components/Results.js:191]:
```suggestion
						<h1>Welcome to Quiz Time</h1>
```

**Review by pendletons — APPROVED** [review]
Looks good - just one question

**Review by cockochan — APPROVED** [review]

File: client/src/Components/Results.js (hunk 2 above)
Comment (naderakhgari) [reply client/src/Components/Results.js:191]: oh it's still remain!
I'll correct it.

File: client/src/Components/Results.js (hunk 1 above)
Comment (naderakhgari) [reply client/src/Components/Results.js:159]: good feedback
It should return 0

**Review by naderakhgari — COMMENTED** [review]
I use the variable attempt number and removed the new variable

**Pull request merged**